testsuite/features/support/constants.rb

```diff
@@ -133,13 +133,22 @@
 # For containers we do not have SCC, so we set the Fake Base Channel
 # for sle_minion
 sle_base_channel =
-  if ENV['PROVIDER'].include? 'podman'
+  if $is_container_provider
     'Fake Base Channel'
+  elsif $product == 'Uyuni'
```

Comment:
Oups, actually it seems we don't have a global variable for that, but just a method "product".
My bad suggesting it sorry.